packages/preset-commonmark/src/mark/strong.ts

```diff
@@ -13,6 +12,7 @@ export const strong = createMark<Keys>((utils) => {
     return {
         id,
         schema: () => ({
+            inclusive: false,
```

Comment:
`inclusive: false` causes the mark to end after one character is entered.

![inclusive](https://user-images.githubusercontent.com/2498173/201256217-63ddba53-db5b-4e92-b653-753587501c62.gif)

Reply:
It's a trade off to support inline sync. We need to provide custom toggle mark command to fix this.

Reply:
Okay, I've created an issue for this. #809